nilearn/datasets.py

```diff
@@ -2568,7 +2568,7 @@ def fetch_abide_pcp(data_dir=None, n_subjects=None, pipeline='cpac',
     pheno = BytesIO(pheno)
     # We enforce a magic string  forcomments because it is 'sharp' by default,
     #   and Python 3.4 doesn't like empty
```

Comment:
@AlexandreAbraham does this comment still make sense? It should be removed if $ is actually used for comments.